core/build/action.yml

```diff
@@ -167,6 +167,36 @@ runs:
         ant -Dcapstone=../../dist removeSource
       shell: bash 
       
+    - name: 'clone v2'
+      run: | 
+        mkdir clone && cd clone
+        git clone https://github.com/zowe/zowe-install-packaging.git -b v2.x/master
+        ZOWE_V2_VERSION=$(cat zowe-install-packaging/manifest.json.template  \
+          | grep version \
+          | head -1 \
+          | awk -F: '{ print $2 }' \
+          | sed 's/[",]//g')
+        ZLUX_CORE_PAX_V2=$(cat zowe-install-packaging/manifest.json.template  \
+          | grep '"org.zowe.zlux.zlux-core": {' -A 10 \
+          | grep '"artifact":' \
+          | grep -m 1 '"artifact":' \
+          | awk -F: '{ print $2 }' \
+          | sed 's/[", ]//g'
+          | sed 's/\.pax$/.tar/')
+        curl https://zowe.jfrog.io/zowe/list/libs-snapshot-local/org/zowe/zlux/zlux-core/${ZOWE_V2_VERSION}-RC/${ZLUX_CORE_PAX_V2} -k -o zlux.tar
+        tar xpoUf zlux.tar 
+        cd zlux-app-manager/system-apps
+        for app_folder in */
+        do 
+          cp -rf $app_folder/web/* ../../../zlux-app-manager/system-apps/$app_folderweb
+        done
+        cd ../..
+        cp -rf zlux-app-manager ../zlux/zlux-app-manager-v2
+        cd ../zlux/zlux-app-manager-v2 && rm -rf bootstrap virtual-desktop/lib
+        cd system-apps && find . -type f -name '*.map' -exec rm {} \;
```

Comment:
you could just delete the entire `system-apps` folder, since the only content that was needed was the `/web` content that was copied out by this time.